app/src/main/java/org/desarrolladorslp/teckersapp/ui/deliverables/DeliverableFragment.kt

```diff
@@ -1,28 +1,54 @@
 package org.desarrolladorslp.teckersapp.ui.deliverables
 
+
+import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.TextView
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.Observer
 import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 import org.desarrolladorslp.teckersapp.R
 
 class DeliverableFragment : Fragment() {
 
-    private lateinit var deliverableViewModel: DeliverableViewModel
+    private lateinit var deliverablesViewModel: DeliverableViewModel
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var viewAdapter: RecyclerView.Adapter<DeliverableAdapter.DeliverableHeaderHolder>
+    private lateinit var viewManager: RecyclerView.LayoutManager
```

Comment:
You probably don't need this to be a property of the fragment.

Reply:
Why?

Reply:
Because you just use it in one place